assets/css/pages/glossary.scss

```
@@ -1,120 +1,117 @@
-a {
-  @extend %font-ui-heading-post-block-title;
-  color: $color-white-180;
-  @include font-size(14px);
-  line-height: rem(18px);
-}
-
-a:active,
-a:hover {
-  color: $color-white-180;
-}
-
-li + li {
-  margin-left: rem(12px);
-}
-
-ol {
-  display: flex;
-  flex-direction: row;
-  justify-content: center;
-  margin: 0;
-  padding: rem(13) 0;
-  @extend %font-ui-heading-post-block-title;
-  color: $color-white-130;
-  @include font-size(14px);
-  line-height: rem(18px);
-}
-
-section {
-  position: relative;
+.glossary {
+  li + li {
+    margin-left: rem(12px);
+  }
 
-  &:not(:last-child)::after {
-    content: '';
-    position: absolute;
-    right: 0;
-    left: 0;
+  ol {
```

The current line height when inspecting on the browser is this:
```line-height: 1.29;```
in the mock up is asking for ```18px``` this is why I was adding the line height like this ```line-height: rem(18px);```
so when is converted we have: ```line-height: 1.125rem```
I just wanted to clarify when is okay to override the line-height?

---

Good question! In this case, we don't want to override it. Typically when a font size or line height has a discrepancy between what is in the placeholders file & what the Figma mockup calls for, it's better to default to the placeholders file, even if it's slightly different from Figma. In this case, the values are really similar, so it's likely that when the Figma files were being cleaned up, that font style was overlooked. So please don't override the line height in this case. :)